### PR TITLE
Stubbing out docs for new editors on managing notifications

### PR DESCRIPTION
**Pull request by arfon**
:wave: @openjournals/joss-editors - I thought it would be useful to add something to our documentation outlining strategies/approaches for managing notifications as a new editor.

If you've developed any approaches yourself, please comment here and I'll extend these docs.

cc-ing a few of our newest editors: @melissawm @kbarnhart @majensen @mjsottile @drvinceknight @meg-simula @oliviaguest @will-rowe

### CODE REVIEW

**Comment by arokem** [comment]
As an editor, you will find yourself writing certain messages again and again. For example, the folllowing message is one that I copy-paste often: 

"""
@ author: At this point, could you please make a new release of the main repo that includes all of the changes that have resulted from the review? Then, please make an archive of the software in Zenodo/figshare/other service and update this thread with the DOI of the archive. For the archive version, please make sure that:

- The title of the archive is the same as the JOSS paper title
- The authors of the archive are the same as the JOSS paper authors
"""

There may be other similar messages that would be useful to collect here.

**Comment by arfon** [comment]
Good idea @arokem - I've just opened a new PR with this response (and others) here: https://github.com/openjournals/joss/pull/619

File: docs/editing.md [hunk 1]
```diff
@@ -216,3 +216,39 @@ Mark yourself as OoO in one of the reviews you're editing in the [joss-reviews](
...
+
+**Unsubscribe from the reviews repository on GitHub**
+
+When you're added to the editorial team on GitHub, you will almost certainly find yourself subscribed (watching) to the [`joss-reviews`](https://github.com/openjournals/joss-reviews) repository. The first thing you should do is set yourself to 'not watching':
+
+![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)
+
+Please note, that by not watching the reviews repository, you will still receive notifications for issues (reviews) where you are `@mentioned`.
+
+**Curate your GitHub notifications experience**
```
Comment (terrytangyuan) [inline docs/editing.md]: Another thing is that if some other editor mentioned all the editors in one review (e.g. asking a particular question) and you think it's not relevant to you, you can manually unsubscribe from that particular review issue.

Comment (leouieda) [reply docs/editing.md]: :+1: for this. Also making sure people use the `joss-eics` instead of `joss-editors` (the names are a bit confusing).

**Pull request merged**